projects/1.18/assets/your-ant-friend/yourantfriend/lang/zh_cn.json

```
@@ -0,0 +1,83 @@
+{
+  "advancements.youarecompletelycrazy.descr": "你肯定是疯了吧？！",
+  "advancements.youarecompletelycrazy.title": "你肯定是疯了吧？！",
+  "block.yourantfriend.anthill_1": "蚁丘",
+  "block.yourantfriend.anthill_2": "蚁丘",
+  "block.yourantfriend.anthill_3": "蚁丘",
+  "block.yourantfriend.anthillcapsule": "蚂蚁饲养工坊",
+  "block.yourantfriend.basket": "野餐篮子",
+  "block.yourantfriend.bricksofdirt": "土制砖头",
+  "block.yourantfriend.dirtplanks": "板制泥土",
+  "block.yourantfriend.pupa": "虫蛹",
+  "effect.yourantfriend.caters_enemy": "小小育虫人之敌",
+  "effect.yourantfriend.strenght": "蚁之力量",
+  "entity.yourantfriend.adult_antlion": "蚁蛉",
+  "entity.yourantfriend.ant": "蚂蚁",
+  "entity.yourantfriend.anteater": "食蚁兽",
+  "entity.yourantfriend.antlion": "蚁狮",
+  "entity.yourantfriend.cater": "小小育虫人",
+  "entity.yourantfriend.invisible": "不可视的",
+  "entity.yourantfriend.weaver_ant": "织叶蚁",
+  "item.yourantfriend.adult_antlion_spawn_egg": "蚁蛉刷怪蛋",
+  "item.yourantfriend.ant_spawn_egg": "蚂蚁刷怪蛋",
+  "item.yourantfriend.anteater_spawn_egg": "食蚁兽刷怪蛋",
+  "item.yourantfriend.antlion_spawn_egg": "蚁狮刷怪蛋",
+  "item.yourantfriend.cater_spawn_egg": "小小育虫人刷怪蛋",
+  "item.yourantfriend.caterhat_boots": "小小育虫人之靴",
+  "item.yourantfriend.caterhat_chestplate": "小小育虫人上衣",
+  "item.yourantfriend.caterhat_helmet": "小小育虫人帽子",
+  "item.yourantfriend.caterhat_leggings": "小小育虫人裤子",
+  "item.yourantfriend.caterpillar": "毛虫",
+  "item.yourantfriend.caterpillarfried": "香酥毛毛虫",
+  "item.yourantfriend.fakeant": "伪造蚂蚁",
+  "item.yourantfriend.gloveofsilk": "真丝手套",
+  "item.yourantfriend.goldencaterpillar": "金毛虫",
+  "item.yourantfriend.invisible_spawn_egg": "不可视的刷怪蛋",
+  "item.yourantfriend.leaf": "叶子",
+  "item.yourantfriend.leafsilk": "叶子",
+  "item.yourantfriend.mango": "芒果",
+  "item.yourantfriend.netheritelionpincers": "下界合金蚁狮钳",
+  "item.yourantfriend.silk": "真丝",
+  "item.yourantfriend.soft_leatherglove": "软革手套",
+  "item.yourantfriend.weaver_ant_spawn_egg": "织叶蚁刷怪蛋",
+  "itemGroup.tabyourantfriendmodtab": "小小养蚁人",
+  "message.follow": "跟上！",
+  "message.go": "走你!",
```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**Go!**
```suggestion
  "message.go": "行动！",
```

---

译法你可以自选，但符号必须是全角“！”
其他就没啥问题了

---

草！没想到在这里出疏漏了xx 现在进行更正【之后勾选刷新拉取请求就可以了吗？

---

[疏忽难免](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/3385#discussion_r1244900230)
commit 修改即可，其他不用管

---

感谢理解。现在进行commit

---

这里没改